components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -499,21 +499,7 @@ public AccessTokenDO getLatestAccessToken(String consumerKey, AuthenticatedUser
 
             prepStmt = connection.prepareStatement(sql);
             prepStmt.setString(1, getPersistenceProcessor().getProcessedClientId(consumerKey));
-            String rootTenantDomain = (String) IdentityUtil.threadLocalProperties.get()
```

Comment:
This logic was added from https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2382/files#diff-6709864b96560546358e0932e0c74fdcdf5dad1ca233d92ac87896c1ea06cf44R501

But later the token revocation logic is improved to set the tenant domain properly. 
https://github.com/wso2-extensions/identity-inbound-auth-oauth/commit/287506fede251231efedfd5baa63eb327904202b#diff-e803b8365d9d99fc10be86bba7d8f5c64ad2c25021e52e8d71e768cc49bf8bf3R743

Hence not required to rely on the thread local, we can get the tenant domain from the authenticated user object itself.